opus/import/main_opus_import.py

```
@@ -418,9 +418,9 @@ def _new_warning_handler(message, category, filename, lineno, file, line):
 
     if not impglobals.ARGUMENTS.dont_use_shelves_only:
         Pds3File.use_shelves_only()
-        # Pds4File.use_shelves_only()    # TODOPDS4
+        Pds4File.use_shelves_only()    # TODOPDS4
     Pds3File.require_shelves(True)
```

Ruff (0.15.7)

[warning] Boolean positional value in function call

(FBT003)